contracts/governance/Staking/Checkpoints.sol

```diff
@@ -79,6 +76,16 @@ contract Checkpoints is StakingStorage, SafeMath96 {
 
     event VestingStakeSet(uint256 lockedTS, uint96 value);
 
+    event TeamVestingCancelled(address indexed caller, address receiver);
+
+    event TeamVestingPartiallyCancelled(
+        address indexed caller,
+        address receiver,
+        uint256 lastProcessedDate
+    );
+
+    event MaxVestingWithdrawIterationsUpdated(uint256 oldMaxIterations, uint256 newMaxIterations);
```

Comment:
missing a test for this event

Reply:
added in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)